### ISSUE TITLE
After pressing Ctrl+Shift+F, you can't Ctrl+V without releasing Ctrl and pressing it again

### ISSUE THREAD

**Issue by albertvaka** [godotengine/godot#90768]
### Tested versions

Happens at least in 4.2.1 on Windows.

### System information

Godot 4.2.1 on Windows

### Issue description

After opening the "Find in files" dialog by pressing Ctrl+Shift+F, pressing Ctrl+V to paste doesn't work if Ctrl has been held down.

### Steps to reproduce

This is easy to reproduce if you press Ctrl+Shift+F to open the "Find in files" dialog, quickly followed by trying to paste with Ctrl+V without releasing Ctrl and pressing it again. You need to release Ctrl once.

### Minimal reproduction project (MRP)

A blank project works.

**Comment by moraguma** [comment]
Managed to reproduce this on 4.2.2 on Windows 10, but not on master. From what I tested, on 4.2.2, using any Ctrl shortcut on another window and switching to a Godot window prevents you from using any Ctrl shortcuts until Ctrl is released. This can be seen when pressing Ctrl+Shift+F on Godot and then trying Ctrl+V in the "Find in files" window, but also when trying to Ctrl+C something from your browser and Ctrl+V it in Godot without releasing Ctrl. This doesn't seem to happen on master.

Here's a video showcasing this behavior on 4.2.2 and on master.

https://github.com/godotengine/godot/assets/63124011/3f4f4a26-bb52-4a5a-a8b4-a9a3282f5753

System info: Godot v4.3.beta (0a83e7c5d) - Windows 10.0.19045 - GLES3 (Compatibility) - NVIDIA GeForce GTX 1650 Ti (NVIDIA; 31.0.15.4633) - Intel(R) Core(TM) i5-10300H CPU @ 2.50GHz (8 Threads)
